blogs/comm-opt/README.md

```diff
@@ -0,0 +1,94 @@
+<div align="center">
+
+# Communictaion Optimization for Large-Scale Training 
+
+</div>
+
+
+## Table of Contents
+1. [Introduction](#introduction)
+2. [Parameter-update All-Reduce Optimization](#ar-opt)
+3. [ZeRO2 Gradient All-Gather Optimization](#ag-opt)
+4. [Sequence-Parallel Optimization](#sp-opt)
+5. [Sequence-Parallel Convergence Consideration](#convergence)
```

Comment:
I think this convergence consideration is out-of-scope for communication optimizations.

Reply:
Even though this is not really part of the optimization, however, we arrive to this issue due to the optimizations that we did. Of course, this is related to the Sequence-Parallelism optimization. So, do you think I can move it to be part of the Ulysses blog post?

Reply:
I have changed a bit the intro to show how this can be relevant in this post.